lib/Sources/Settings/SettingsView.swift

```diff
@@ -75,7 +76,14 @@ public struct SettingsView: View {
                 .listRowTintedButton(color: Color.topaz800) {
                     model.clearCacheButtonTapped()
                 }
+                .sheet(isPresented: $model.presentClearCacheDialogue, content: {
```

Comment:
Why is this a sheet instead of a `confirmationDialog`?

Reply:
Sheesh. Yeah, sorry. Just totally missed that this exists. Will only take a few minutes to switch around.